sound/soc/codecs/rt715-sdca.c

```diff
@@ -311,6 +437,26 @@ static int rt715_sdca_put_volsw(struct snd_kcontrol *kcontrol,
 	.private_value = SOC_DOUBLE_R_VALUE(reg_left, reg_right, xshift, \
 					    xmax, xinvert) }
 
+#define RT715_SDCA_EXT_TLV(xname, reg_base, xhandler_get,\
+	 xhandler_put, tlv_array) \
+{	.iface = SNDRV_CTL_ELEM_IFACE_MIXER, .name = (xname), \
+	.access = SNDRV_CTL_ELEM_ACCESS_TLV_READ | \
+		 SNDRV_CTL_ELEM_ACCESS_READWRITE, \
+	.tlv.p = (tlv_array), \
+	.info = rt715_sdca_vol_info, \
+	.get = xhandler_get, .put = xhandler_put, \
+	.private_value = reg_base }
+
+#define RT715_SDCA_BOOST_EXT_TLV(xname, reg_base, xhandler_get,\
+	 xhandler_put, tlv_array) \
+{	.iface = SNDRV_CTL_ELEM_IFACE_MIXER, .name = (xname), \
+	.access = SNDRV_CTL_ELEM_ACCESS_TLV_READ | \
+		 SNDRV_CTL_ELEM_ACCESS_READWRITE, \
+	.tlv.p = (tlv_array), \
+	.info = rt715_sdca_boost_info, \
+	.get = xhandler_get, .put = xhandler_put, \
+	.private_value = reg_base }
```

Comment:
I guess strictly speaking in both above macro definitions you need to put all macro parameters in parentheses?
```
	.get = (xhandler_get), .put = (xhandler_put), \
	.private_value = (reg_base) }
```

Reply:
> I guess strictly speaking in both above macro definitions you need to put all macro parameters in parentheses?
> 
> ```
> 	.get = (xhandler_get), .put = (xhandler_put), \
> 	.private_value = (reg_base) }
> ```
why should these callback and reg_base need to be put in parentheses?

Reply:
@jack-cy-yu for the same reason it's done for other macto parameters. But assignment already has quite a low priority, so it's unlikely to cause any problems, just looks a bit strange.